scripts/node.sh

```diff
@@ -308,6 +308,20 @@ verify_checksum() {
    return 0
 }
 
+verify_signature() {
+   local dir file 
+   dir="${1}"
+   file="${dir}/${2}"
+   sigfile="${dir}/${2}.sig"
+
+   result=$(openssl dgst -sha256 -verify "${outdir}/harmony_pubkey.pem" -signature "${sigfile}" "${file}" 2>&1)
```

Comment:
openssl so buggy/possible to have bad version. Can we have a known version of openssl that we use this against?